protocol/messenger_communities.go

```diff
@@ -792,12 +792,16 @@ func (m *Messenger) requestCommunityInfoFromMailserver(communityID string) (*com
 		nil,
 		nil,
 		filter,
-		true)
+		waitForResponse)
 
 	if err != nil {
 		return nil, err
 	}
 
+	if !waitForResponse {
+		return nil, nil
```

Comment:
In this case, the filter for this community will not be removed after the community is fetched. Is that desired behavior?